content/docs/reference-react.md

```diff
@@ -88,6 +92,42 @@ If your React component's `render()` function renders the same result given the
 
 * * *
 
+### `React.pure`
+
+```javascript
+const MyComponent = React.pure(function(props) {
+  /* render using props */
+});
+```
+
+`React.pure` is a [higher order component](/docs/higher-order-components.html). It's similar to [`React.PureComponent`](#reactpurecomponent) but for function components instead of classes.
```

Comment:
```suggestion
`React.memo` is a [higher order component](/docs/higher-order-components.html). It's similar to [`React.PureComponent`](#reactpurecomponent) but for function components instead of classes.
```